end2end/tests-new/hono-pg-ts-esm.test.mjs

```diff
@@ -147,3 +150,86 @@ test("it does not block request in monitoring mode", async () => {
     server.kill();
   }
 });
+
+test("It reports own http requests in heartbeat events", async () => {
+  const response = await fetch(`${testServerUrl}/api/runtime/apps`, {
+    method: "POST",
+  });
+  const body = await response.json();
+  const token = body.token;
+
+  const server = spawn(
+    `node`,
+    [
+      "--require",
+      "@aikidosec/firewall/instrument",
+      "--experimental-strip-types",
+      "./app.ts",
+      port3,
+    ],
+    {
+      cwd: pathToAppDir,
+      env: {
+        ...process.env,
+        AIKIDO_TOKEN: token,
+        AIKIDO_ENDPOINT: testServerUrl,
+        AIKIDO_DEBUG: "true",
+      },
+    }
+  );
+
+  try {
+    server.on("error", (err) => {
+      fail(err);
+    });
+
+    let stdout = "";
+    server.stdout.on("data", (data) => {
+      stdout += data.toString();
+    });
+
+    let stderr = "";
+    server.stderr.on("data", (data) => {
+      stderr += data.toString();
+    });
+
+    // Wait for the server to start
+    await timeout(2000);
+
+    await fetch(`http://127.0.0.1:${port3}/`, {
+      method: "GET",
+      signal: AbortSignal.timeout(5000),
+    });
+
+    // Wait for heartbeat to be sent
+    await timeout(35000);
+
+    const eventsResponse = await fetch(`${testServerUrl}/api/runtime/events`, {
+      method: "GET",
+      headers: {
+        Authorization: token,
+      },
+      signal: AbortSignal.timeout(5000),
+    });
+
+    const events = await eventsResponse.json();
+    const heartbeatEvents = events.filter(
+      (event) => event.type === "heartbeat"
+    );
+    equal(heartbeatEvents.length, 1);
+
+    const heartbeatEvent = heartbeatEvents[0];
+
+    equal(heartbeatEvent.hostnames.length, 1);
+
+    const hostname = heartbeatEvent.hostnames[0];
+
+    equal(hostname.hostname, "localhost");
+    equal(hostname.hits, 2);
+    equal(hostname.port, 5874);
```

Comment:
and the real time polling happens after 1 minute, extend waiting time? So 3 hits?

(but it will be the same hostname + port though)